src/fontra/views/editor/visualization-layer-definitions.js

```diff
@@ -984,20 +984,21 @@ registerVisualizationLayerDefinition({
   zIndex: 490,
   screenParameters: {
     strokeWidth: 1,
-    originMarkerRadius: 4,
+    anchorRadius: 4,
   },
-  colors: { color: "#AAA8" },
-  colorsDarkMode: { color: "#8888" },
+  colors: { color: "#AAA8", colorAnchor: "#00AAA820" },
```

Comment:
I think you are also changing the hue here. I would expect the anchorColor to be a variant of #AAA8, say #AAA4 (ie. just changing the opacity). Same for dark mode.

(Note that #AAA8 is a shortcut for #AAAAAA88)

Reply:
Thanks for the explanation. I changed the colors. But it seems the blue in dark mode does not get any transparency and I don't know why. I tried several different values.